modules/cudf/src/series.ts

```diff
@@ -627,6 +627,59 @@ export class AbstractSeries<T extends DataType = any> {
     return this.__construct(this._col.gather(selection._col));
   }
 
+  /**
+   * Returns the first n rows.
+   *
+   * @param n The number of rows to return.
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   *
+   * const a = Series.new([4, 6, 8, 10, 12, 1, 2]);
+   * const b = Series.new(["foo", "bar", "test"]);
+   *
+   * a.head(); // [4, 6, 8, 10, 12]
+   * b.head(1); // ["foo"]
+   * a.head(-1); // throws index out of bounds error
+   * ```
+   */
+  head(n = 5): Series<T> {
+    if (n < 0) { throw new Error('Index provided is out of bounds'); }
+    const selection = Series.new({
+      type: new Int32,
+      data: Array.from({length: n < this._col.length ? n : this._col.length}, (_, i) => i)
+    });
```

Comment:
I think [`Series.sequence`](https://github.com/rapidsai/node-rapids/blob/main/modules/cudf/src/series.ts#L785) could be used here (and a above)

Reply:
Neat.